field/Cargo.toml

```diff
@@ -10,6 +10,10 @@ repository.workspace = true
 keywords.workspace = true
 categories.workspace = true
 
+[features]
+default = []
+wasm_opt = []
```

Comment:
new feature used to enable/disable the new goldilocks mul implementation, regardless of the target architecture